### ISSUE TITLE
Can't Dismiss Mercenaries And They Take Items From Chests

### ISSUE THREAD

**Issue by peachycats** [ldtteam/minecolonies-features#231]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version: 1.16.4
- MineColonies Version: 0.13.547-ALPHA
- Related Mods and their Versions:

## Is your feature request related to a problem?
Kind of? 

## Enhancement or Change description
Two issues, one is that you cannot get rid of mercenaries manually. I hired them to remove some pillagers and then they ran around, not despawning and were taking items out of the builder's hut while I'm trying to build. I'd like for there to be a button underneath the 'hire mercenaries' that allows us to dismiss them once we're happy with them. The pillagers taking items from chests from what I was told isn't supposed to happen, they should just take anything that's dropped.

### Notes or related things
That's about it lol. 

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by ravenbuilder934** [comment]
No, mercenaries taking items from chests/etc is intentional

**Comment by Deightine** [comment]
> It would be nice to have a way to dismiss them though

Indeed. Right now (as is historically accurate, unfortunately), its like a mouse inviting in a cat to deal with their rat problem. The cat isn't going to just wander off after. Mercenaries tend to be kind of rough.

So with that in mind, I think I'd enjoy if the way of getting rid of the mercenaries was a bit more direct. Like they spawn with a leader who hangs out in your town hall and you have to make him leave. Either with another payment (a bribe), to go peacefully, or with a one-on-one fight. Either way, the rest of the mercenaries would leave.

**Comment by jzisser9** [comment]
> > It would be nice to have a way to dismiss them though
> 
> Indeed. Right now (as is historically accurate, unfortunately), its like a mouse inviting in a cat to deal with their rat problem. The cat isn't going to just wander off after. Mercenaries tend to be kind of rough.
> 
> So with that in mind, I think I'd enjoy if the way of getting rid of the mercenaries was a bit more direct. Like they spawn with a leader who hangs out in your town hall and you have to make him leave. Either with another payment (a bribe), to go peacefully, or with a one-on-one fight. Either way, the rest of the mercenaries would leave.

I think these are great ideas. Mercenaries are meant to come with drawbacks, so being able to just dismiss them outright seems too easy.

**Comment by Deightine** [comment]
> I think these are great ideas. Mercenaries are meant to come with drawbacks, so being able to just dismiss them outright seems too easy.

That was my thought as well. Rather than just smack a button and get nothing out of it, it leaves you with the decision: "Do I try to kill all of these invaders myself and chance being overrun? Or do I get the mercenaries? But if I get the mercenaries, I might not be able to afford to pay them to leave... And that means either I let them steal everything or I have to fight their boss..."

Sometimes its easy for folks to forget that games are supposed to have obstacles in them, else they aren't really games anymore.
